src/views/LostItems/LostItemsListContainer/components/LostItemsList/components/RecordStatus/RecordStatus.js

```diff
@@ -12,7 +12,7 @@ import {
 } from '../../../../../constants';
 import { getRecordStatus } from '../../util';
 import DateTimeFormatter from '../DateTimeFormatter';
-import { useLocalizeCurrency } from '../../../../../../../hooks/useLocalizedCurrency/useLocalizeCurrency';
+import useLocalizeCurrency from '../../../../../../../hooks/useLocalizedCurrency/useLocalizeCurrency';
```

Comment:
after updating src/hooks/index.js file on the lines of other hooks exports (as suggested below in one of the comments), adjust this import